src/scratchOrgInfoGenerator.ts

```diff
@@ -208,32 +208,32 @@ export const generateScratchOrgInfo = async ({
 /**
  * Returns a valid signup json
  *
- * @param definitionjson
- * @param definitionfile
- * @param connectedAppConsumerKey
- * @param durationdays
- * @param nonamespace
- * @param noancestors
- * @param orgConfig
+ * @param definitionjson org definition in JSON format
+ * @param definitionfile path to an org definition file
+ * @param connectedAppConsumerKey The connected app consumer key. May be null for JWT OAuth flow.
+ * @param durationdays duration of the scratch org (in days) (default:1, min:1, max:30)
+ * @param nonamespace create the scratch org with no namespace
+ * @param noancestors do not include second-generation package ancestors in the scratch org
+ * @param orgConfig overrides definitionjson
  * @returns scratchOrgInfoPayload: ScratchOrgInfoPayload;
             ignoreAncestorIds: boolean;
             warnings: string[]; 
  */
 export const getScratchOrgInfoPayload = async (options: {
-  definitionjson: string;
-  definitionfile: string;
-  connectedAppConsumerKey: string;
+  definitionjson?: string;
```

Comment:
suggestion: consider using the TS Pick to avoid having to repeat the types for all these properties, since this is really a subset of `ScratchOrgCreateOptions`